reporter/src/main/kotlin/reporters/freemarker/asciidoc/AsciiDocTemplateReporter.kt

```diff
@@ -58,7 +58,7 @@ abstract class AsciiDocTemplateReporter(private val backend: String, override va
         ASCII_DOC_FILE_EXTENSION,
         ASCII_DOC_TEMPLATE_DIRECTORY
     )
-    private val asciidoctor = Asciidoctor.Factory.create()
+    private val asciidoctor by lazy { Asciidoctor.Factory.create() }
```

Comment:
A short Google search seems to indicate that _Asciidoctor_ is thread-safe, e.g. [1]. So, do you think it would be possible to use a single shared instance for all reporter instances?

[1] https://github.com/asciidoctor/asciidoctor-maven-plugin/issues/326

Reply:
That might be an option, I think we would have to give it a try.